src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -394,6 +382,23 @@ public async Task LoadAsync()
             }
         }
 
+        private void AddTotpGroupItem(bool canAccessPremium, List<GroupingsPageListGroup> groupedItems, bool uppercaseGroupNames)
+        {
+            if (canAccessPremium && (TOTPCiphers?.Any() ?? false))
```

Comment:
I think that `(TOTPCiphers?.Any() ?? false)` can be written as `TOTPCiphers?.Any() == true` to be less confusing to read and you can remove the parenthesis